No multiple parents (#1279)
radiantearth/stac-spec: item-spec/common-metadata.md

| parent     | URL to the parent STAC entity ([Catalog](../catalog-spec/README.md) or [Collection](../collection-spec/README.md)). | application/json                                     |
| collection | URL to the parent Collection. *Absolute* URLs should be used whenever possible.                                     | application/json                                     |
| child      | URL to a child STAC entity ([Catalog](../catalog-spec/README.md) or [Collection](../collection-spec/README.md)).    | application/json                                     |
| item       | URL to a STAC Item.                                                                                                 | application/geo+json (preferred) or application/json |

#### Self relation

The `self` relation is used to link to the STAC entity itself.
This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from.

#### Root and parent relation

> The `root` and `parent` relations are used to link to the root and parent STAC entity, which is either a [Catalog](../catalog-spec/README.md) or a [Collection](../collection-spec/README.md).
> **ONLY ONE** link with `root` or `parent` relationship and `application/json` type is allowed.
> Other version of the `root` or `parent` link can be added with different `type` field as long as they point the same conceptual entity (e.g. identified by the ID).

Reviewer: ```suggestion
The `root` and `parent` relations are used to link to the root and parent STAC entity,
which is either a [Catalog](../catalog-spec/README.md) or a [Collection](../collection-spec/README.md).
Conceptually, STAC entities SHALL have no more than one parent entity.
As such, STAC entities also can have no more than one root entity.
Therefore, there's usually just one link with `root` or `parent` relationship
unless different variations of the same conceptual entity exist (identified by the ID).
Different variations could be:
- a different encoding (see the `type` property), e.g. a HTML version in addition to JSON
- a different language (see the `hreflang` property). e.g. a German version in addition to English
```

Trying to clarify a bit more and add that different languages could also be a variations.

#### Collection relation

The referenced Collection is STRONGLY RECOMMENDED to implement the same STAC version as the Collection.

> A link with this `rel` type is *required* for STAC item if the `collection` field in properties is present.

Reviewer: This line should probably be in the Item Spec where the collection field is defined.

#### Item relation

> All Items linked from a Collection MUST refer back to its Collection with the [`collection` relation type](#collection-relation).

Reviewer: This seems a bit off, i.e. there's context missing. The header is Item reltation, but it discussed the collection relation.

### Additional relations

| Type         | Description                                                                                                      | Media Type       |
| ------------ | ---------------------------------------------------------------------------------------------------------------- | ---------------- |
| license      | The license URL(s) for the Item SHOULD be specified if the `license` field is **not** a SPDX license identifier. | Any              |

> | derived_from | URL to a STAC Entity that was used as input data in the creation of this Entity.                                 | application/json |

Reviewer: Maybe we should move the derived_from to the best practices and license back to the common metadata field. But that doesn't need to be part of this PR necessarily.

PlanetScope Ortho Tile Product has an `gsd` of 3.7 (or 4 if rounding), even though the pixel size of the images is 3.125.
For example, one might choose for WorldView-2 the Multispectral 20° off-nadir value of 2.07
and for WorldView-3 the Multispectral 20° off-nadir value of 1.38.

## Relation types

STAC Entities use a variety of `rel` types in the link object,
to describe the exact nature of the link between the STAC object and the entity it is linking to.
It is recommended to use the official
[IANA Link Relation Types](https://www.iana.org/assignments/link-relations/link-relations.xhtml) where possible.

> ### STAC Structure relation

Reviewer: ```suggestion
### Hierarchical relations
```

| self       | *Absolute* URL to the location that the STAC file can be found online, if available.                                | application/json                                     |
| root       | URL to the root STAC entity ([Catalog](../catalog-spec/README.md) or [Collection](../collection-spec/README.md)).   | application/json                                     |
| parent     | URL to the parent STAC entity ([Catalog](../catalog-spec/README.md) or [Collection](../collection-spec/README.md)). | application/json                                     |
| child      | URL to a child STAC entity ([Catalog](../catalog-spec/README.md) or [Collection](../collection-spec/README.md)).    | application/json                                     |
| collection | URL to the parent Collection. *Absolute* URLs should be used whenever possible.                                     | application/json                                     |
| item       | URL to a STAC Item.                                                                                                 | application/geo+json (preferred) or application/json |

> #### Self relation

Reviewer: ```suggestion
#### `self` relation
```

The `self` relation is used to link to the STAC entity itself.
This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from.

> #### Root and parent relation

Reviewer: ```suggestion
#### `root` and `parent` relation
```

The `root` and `parent` relations are used to link to the root and parent STAC entity,
which is either a [Catalog](../catalog-spec/README.md) or a [Collection](../collection-spec/README.md).
Conceptually, STAC entities SHALL have no more than one parent entity.
As such, STAC entities also can have no more than one root entity.
Therefore, there's usually just one link with `root` or `parent` relationship
unless different variations of the same conceptual entity exist (identified by the ID).
Different variations could be:

- a different encoding (see the `type` property), e.g. a HTML version in addition to JSON
- a different language (see the `hreflang` property). e.g. a German version in addition to English

> #### Child relation

Reviewer: ```suggestion
#### `child` relation
```

The `child` relation is **ONLY** used to link a catalog or collection to a child catalog or collection.

> #### Collection and item relation

Reviewer: ```suggestion
#### `collection` and `item` relation
```
